broadcasting/10_RTSP_Pipeline.md

```diff
@@ -0,0 +1,89 @@
+As explained in the [Architecture chapter](02_Architecture.md), the pipeline will consist of a couple of elements, that will be processing the RTP stream.
+
+The flow of the pipeline will consist of three steps. First, when the pipeline is initialized we will start the Connection Manager, which will set up the RTP stream via the RTSP.
+Once that is finished, we will initialize the first elements in the pipeline - the `UDP Source` and `RTP SessionBin`, which will allows us to receive RTP packets and process them.
+When the SessionBin detects that the RTP stream has been started it will notify the pipeline with the `:new_rtp_stream` notification we will add the remaining elements to the pipeline, allowing for the whole conversion process to take place.
```

Comment:
```suggestion
When the SessionBin detects that the RTP stream has been started, it will notify the pipeline with the `:new_rtp_stream` notification. Later on, we will add the remaining elements to the pipeline, allowing for the whole conversion process to take place.
```

Reply:
done